api.js

```diff
@@ -713,26 +713,41 @@ exports.deleteUser = function (userId) {
 exports.meetingSuggestion = function(obj) {
 
     var participants = obj.participants || [];
+    var resources = obj.resources || [];
+    var duration = obj.duration || "1H";
 
     //using ADDDATE(CURDATE(), 4) just to limit the amount of meetings to search through later
-    const getRoomMeetingsSql = "SELECT room_name, GROUP_CONCAT(DISTINCT start_datetime, '|', end_datetime SEPARATOR '|') AS meetingTimes FROM ebdb.Meeting "
-                      + "WHERE start_datetime >= CURDATE() AND end_datetime <= ADDDATE(CURDATE(), 4) GROUP BY room_name;";
-    const getOtherRoomsSql = "SELECT name FROM ebdb.MeetingRoom WHERE name NOT IN (SELECT DISTINCT room_name FROM ebdb.Meeting "
-                           + "WHERE start_datetime >= CURDATE() AND end_datetime <= ADDDATE(CURDATE(), 4));"
-    const getMeetingSql = "SELECT DISTINCT start_datetime, end_datetime FROM ebdb.Meeting WHERE start_datetime >= CURDATE() AND end_datetime <= ADDDATE(CURDATE(), 4) AND "
-                        + "calendar IN (SELECT primary_calendar FROM ebdb.User WHERE email IN (?)) ORDER BY end_datetime;";
+    var getRoomMeetingsSql = "SELECT room_name, GROUP_CONCAT(DISTINCT start_datetime, '|', end_datetime SEPARATOR '|') AS meetingTimes FROM ebdb.Meeting "
```

Comment:
Based on an earlier update I think the SQL will need the rooms to be capitalized correctly (e.g. ebdb.MeetingRoom instead of ebdb.meetingroom, etc.)  

...It may be a config or mysql version issue, but correcting the capitalization fixed an issue with the previous update.

Reply:
done